_includes/alert-ip-address.md

```diff
@@ -0,0 +1,4 @@
+{% include alert.html type="warning" icon="warning" header="Differing IP
+addresses" body="Please note that the callback is sent from `91.132.170.1` in
+the <strong>test environment</strong> and from `82.115.146.1` in the
```

Comment:
```suggestion
the <strong>external integration environment</strong> and from `82.115.146.1` in the
```

Reply:
Sorry, I can see that we call "external integration" for "test environment". Ignore this.